drivers/pinmux/stm32/pinmux_stm32l4x.h

```diff
@@ -52,6 +54,8 @@
 	(STM32_PINMUX_ALT_FUNC_8 | STM32_OPENDRAIN_PULLUP)
 #define STM32L4X_PINMUX_FUNC_PA6_ADC12_IN11 STM32_MODER_ANALOG_MODE
 
+#define STM32L4X_PINMUX_FUNC_PA7_I2C3_SCL                                      \
```

Comment:
Specific to STM32L496 soc variant. Should be under #ifdef

Reply:
Done.